kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/utils/TestUtils.scala

```diff
@@ -102,6 +104,10 @@ trait TestUtils extends Assertions with SQLHelper {
     }
   }
 
+  implicit object ResourceLoader {
+    lazy val classLoader: ClassLoader = ResourceLoader.getClass.getClassLoader
+  }
+
```

Comment:
why do you need this? Why not just use `getClass.getClassLoader`?

Reply:
@vkorukanti sorry, I don't really understand what you mean, `ResourceLoader` is an object/class not a method. If just use `getClass.getClassLoader` each time we use it, will look like `ResourceLoader.getClass.getClassLoader` (looks a bit odd).
I needed the `classLoader` field to make it easier to use `ResourceLoader.classLoader` and I used `lazy` because I thought it might be accessed multiple times.

Reply:
I found a similar implementation at [GoldenTableUtils.scala](https://github.com/delta-io/delta/blob/efc0e34dd907e257f84eabb8d43f3e0346859cc9/connectors/golden-tables/src/main/scala/io/delta/golden/GoldenTableUtils.scala#L23)

Reply:
Sorry, what I meant is: whenever we are getting a class loader, basically get the current object class with `getClass()` (eg. `TestUtils.getClass()`) and get the class loader from class object. I didn't understand why `ResourceLoader.getClass` is used. Is there anything special about it?